examples/guided_onboarding_example/public/components/step_two.tsx

```diff
@@ -26,18 +25,18 @@ export const StepTwo = (props: StepTwoProps) => {
   const {
     guidedOnboarding: { guidedOnboardingApi },
   } = props;
-  const { search } = useLocation();
-  const history = useHistory();
-
-  const query = React.useMemo(() => new URLSearchParams(search), [search]);
-  useEffect(() => {
-    if (query.get('showTour') === 'true') {
-      setIsTourStepOpen(true);
-    }
-  }, [query]);
 
   const [isTourStepOpen, setIsTourStepOpen] = useState<boolean>(false);
 
+  useEffect(() => {
+    const subscription = guidedOnboardingApi
+      ?.isGuideStepActive$('search', 'search_experience')
```

Comment:
I this this may need to change to `browse_docs` based on the changes I made in https://github.com/elastic/kibana/pull/141233/files